cmd/utils/flags.go

```diff
@@ -789,20 +789,40 @@ func SetP2PConfig(ctx *cli.Context, cfg *p2p.Config) {
 	setBootstrapNodes(ctx, cfg)
 	setBootstrapNodesV5(ctx, cfg)
 
+	lightClient := ctx.GlobalBool(LightModeFlag.Name) || ctx.GlobalString(SyncModeFlag.Name) == "light"
+	lightServer := ctx.GlobalInt(LightServFlag.Name) != 0
+	lightPeers := ctx.GlobalInt(LightPeersFlag.Name)
+
 	if ctx.GlobalIsSet(MaxPeersFlag.Name) {
 		cfg.MaxPeers = ctx.GlobalInt(MaxPeersFlag.Name)
+	} else {
```

Comment:
Hmm... Looks like if you specify `max-peers`, it's not possible to also have lightpeers.

Reply:
It still looks to me like it is entirely possible. ETH peer limit is calculated here:
https://github.com/ethereum/go-ethereum/pull/16010/files#diff-e42690942282286ede9e53bbe7b0c248R399
If you explicitly specify maxpeers, ethpeers is still going to be maxpeers-lightpeers (if LES service is enabled, of course). If you don't, the default value is increased by lightpeers so that the number of ETH peers does not change because of enabling server mode.

Reply:
Yep, tried it now, specified maxpeers=150 in server mode and I got ETH=50 LES=100 total=150.